app/src/test/java/mozilla/lockbox/presenter/LockedPresenterTest.kt

```diff
@@ -38,10 +38,10 @@ class LockedPresenterTest {
 
     open class FakeFingerprintStore : FingerprintStore() {
         override var fingerprintManager: FingerprintManager? =
-            RuntimeEnvironment.application.getSystemService(Context.FINGERPRINT_SERVICE) as? FingerprintManager
+            ApplicationProvider.getApplicationContext<Context>().getSystemService(Context.FINGERPRINT_SERVICE) as? FingerprintManager
```

Comment:
ApplicationProvider.getApplicationContext() can be pulled out into a context val since it's called multiple times.